src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -479,9 +479,9 @@ internal IWebUI CreateWebAuthenticationDialog(UIParent parent, Prompt prompt, Re
 
 #if WINDOWS_APP || DESKTOP
             //hidden webview can be used in both WinRT and desktop applications.
-            parent.UseHiddenBrowser = prompt.Equals(Prompt.Never);  // todo(migration): what to do here now that Prompt.Never is gone?
+            parent.UseHiddenBrowser = prompt.Equals(Prompt.Never);
```

Comment:
>parent.UseHiddenBrowser = prompt.Equals(Prompt.Never); [](start = 12, length = 54)

this scenario is interesting. @JM, should we continue to support the hidden browser?